### ISSUE TITLE
Trader Dupe.

### ISSUE THREAD

**Issue by DevSkoll** [CitizensDev/Citizens#651]
You probably already know this, but if u try to but an item you cant aford by clicking on it really fast and pressing e almost instantly, you get to drop that item out of the traders inventory.

**Comment by DevSkoll** [comment]
I have a suggestion to this. You can set a block radius around a trader to remove any drops, like world edits code for
 /remove drops. but only have that active while someone is shopping and 2 seconds after a person is done shopping.

**Comment by eBeeT** [comment]
Yeah I noticed that sometimes items from shop disappears... I think players bugging.
